bootstrap.sh

```diff
@@ -22,6 +22,41 @@ function job_cleanup() {
     rm -fr $TMP_JOB_DIR
 }
 
+get_image_tag() {
+
+    local image_tag
+    local git_short_hash
+
+    git_short_hash=$(git rev-parse --short=7 HEAD)
+
+    if is_change_request; then
+        image_tag="pr-$(get_change_request_id)-${git_short_hash}"
+    else
+        image_tag="$git_short_hash"
+    fi
+
+    echo "$image_tag"
+}
+
+is_change_request() {
+  [ -n "$ghprbPullId" ] || [ -n "$gitlabMergeRequestId" ]
```

Comment:
I thought one of those variables were always set so wouldn't this always resolve to `true`?

Reply:
not if you run it outside of a "change request" context, like running the pr-check locally, which I did to test it :)

Reply:
also the bootstrap script runs not only in "pr_checks" - which may lead to pipeline runs that are not in a "change request context"

Reply:
Ohhh I see.